src/components/Trends/TrendsTable/TrendsTable.js

```diff
@@ -3,14 +3,17 @@ import Table from 'react-table'
 import cx from 'classnames'
 import { Link } from 'react-router-dom'
 import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
 import {
   Label,
   Checkbox,
   PanelWithHeader,
   Panel,
   Icon,
-  Tooltip
+  Tooltip,
+  Button,
```

Comment:
@DmitriVanGuard we are not python. What is the new trend to use "," at the end?

Reply:
Prettier

Reply:
Oh, yes. Don't notify it before :)